CHANGELOG.md

```diff
@@ -4,14 +4,18 @@ Since v0.10.0 this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Upcoming version
 
+No changes yet
```

Comment:
For the record, we might add a period here, or remove it from the GOVERNANCE.md template, depending on the outcome of [this discussion](https://github.com/hamstar/Wikimate/pull/105#discussion_r663699913).

Reply:
I'm okay with the addition.

Reply:
[Added](https://github.com/hamstar/Wikimate/compare/906e7ae7511a9c42ec0f856efa518542dc820cf0..c4ab5d93ee8d42f07b9eb8f8af4a78adc46eb9b5).